lib/ui/platform_dispatcher.dart

```diff
@@ -1333,7 +1340,13 @@ class ViewConfiguration {
   /// relative to.
   ///
   /// If null, then this configuration represents a top level view itself.
-  final FlutterView? window;
+  @Deprecated('''
+      Use the `view` property instead.
+  ''')
+  FlutterView? get window => _view;
+  FlutterView? get view => _view;
```

Comment:
Add doc to `view`.

Reply:
Added in 56e558cca6d26765dac6ac5bf815074f04d30bca, it's just a duplicate of the doc for `window` since they are both the same.